plugins/cost-insights/src/components/CostInsightsPage/CostInsightsPage.tsx

```diff
@@ -94,26 +103,32 @@ const CostInsightsPage = () => {
           dispatchLoadingInsights(true);
           const [
             fetchedProjects,
-            fetchedCosts,
             fetchedAlerts,
+            fetchedMetricData,
+            fetchedCosts,
           ] = await Promise.all([
             client.getGroupProjects(pageFilters.group),
+            client.getAlerts(pageFilters.group),
+            pageFilters.metric
+              ? client.getMetricData(
+                  pageFilters.metric,
+                  intervalsOf(pageFilters.duration),
```

Comment:
We can probably set the intervals to a `const` to improve the readability of this long chunk